app/forms/work_form.rb

```diff
@@ -4,6 +4,12 @@
 class WorkForm < ApplicationForm
   accepts_nested_attributes_for :related_links
 
+  def initialize(params = {})
+    # TODO: Extract nested form handling. See https://github.com/sul-dlss/hungry-hungry-hippo/issues/218
```

Comment:
Itd be helpful to know how the current nested attrs impl is *not* doing the right thing for publication dates. What all is broken?

Reply:
Nevermind, since you're already doing the extraction! 🎉